kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/SchemaUtils.java

```diff
@@ -436,6 +441,138 @@ private static Map<Long, StructField> fieldsById(StructType schema) {
     return columnIdToField;
   }
 
+  /* Validates that array and map types have valid nested IDs
```

Comment:
Sorry -- please follow the method-doc convetion used throughout kernel. Nit comment but it is important for readability that we follow a consistent code style throughout the code base.

Mehtod docs use

```java
  /**
   *
   */
```

Reply:
No problem, just noticed this and fixed it and in another location